LiskHQ/lisk-hub
# Finalize hw manager - Closes #2446

### What issue have I solved?

Fixes #2446 
Fixes #1954

### How have I implemented/fixed it?

- [x] 🔥 Remove hwWallets reducer
- [x] 🔥 Remove isHarwareWalletConnected from settings store
- [x] Delete other obsolete files
- [x] Remove unncessary code from existing hwManager files
- [x] :bug: Fix voting cancelation


Note that this branch contains all changes from the whole epic. To see just changes new in this PR, see https://github.com/LiskHQ/lisk-hub/compare/1954-hardware-wallet-manager...2446-finalize-hw-manager

### How has this been tested?


Check for regressions in supported hardware wallets:
 - connect (also without open app for Ledger)
 - login
- send (including cancel and retry)
- vote (including cancel)
- disconnect

Check all of the above with:
- Ledger Nano S
- Ledger Nano X
- Trezor Model T


### Review checklist
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)
